src/transforms/push.ts

```diff
@@ -172,7 +178,10 @@ export const processPush = async (github: GitHubAPI, payload) => {
 				updateSequenceId: Date.now()
 			};
 
-			await jiraClient.devinfo.repository.update(jiraPayload);
+			const webhook = await jiraClient.devinfo.repository.update(jiraPayload);
+			const webhookName = payload.name || "none";
+
+			calculateProcessingTimeInSeconds(webhookReceived, webhookName, webhook?.status);
```

Comment:
If webhookReceived is "none" it will fail here. We need to add a check if we have time here